flask_python_ldap/__init__.py

```diff
@@ -114,9 +114,17 @@ def base(self, base):
         self._base = base
         return self
 
+    def get(self, dn):
```

Comment:
To get the DN it would be better to [search for the DN](http://www.python-ldap.org/en/latest/reference/ldap.html#ldap.LDAPObject.search_ext_s) using the full DN, SCOPE_BASE and no filter. This also means you do not need to do any string splitting.
```python
search_ext_s(dn, SCOPE_BASE, ...)
```

Reply:
Ok. thank you